docs/core/install/macos.md

```diff
@@ -170,6 +173,7 @@ Visual Studio for Mac installs the .NET SDK when the **.NET** workload is select
 
 | .NET SDK version      | Visual Studio version                                |
 | --------------------- | ---------------------------------------------------- |
+| 8.0                   | Visual Studio 2022 for Mac 17.8 or higher.           |
```

Comment:
Hello @gewarren ,

My client is looking to migrate a large scale Xamarin.Native app to .NET style. For IDE we are using Visual Studio for Mac 2022. Noticed that in this document, it mentioned .NET 8 will be supported by Visual Studio 2022 for Mac 17.8 or higher.

Currently Visual Studio for Mac 2022 is up to `17.6.8` (Release Notes https://learn.microsoft.com/en-us/visualstudio/releases/2022/mac-release-notes#17.6.8)

Does it mean there will be a `17.8` for macOS version to be released before the it's retirement in Auguest 2024? (Visual Studio for Windows does have 17.8 released to support .NET 8)

Thanks

Reply:
@thomaschenato Sorry, you are correct. Based on https://dotnet.microsoft.com/en-us/download/dotnet/8.0, it looks like version 17.6 is enough for .NET 8. I'll update this page.

Reply:
@gewarren Thank you for the response.

I have Visual Studio for Mac 2022 17.6.9 installed (Released on Feb 13 2024), it doesn't indicate it will support .NET 8 SDKs. Release notes: https://learn.microsoft.com/en-us/visualstudio/releases/2022/mac-release-notes#1769--visual-studio-2022-version-1769-newreleasebutton

<img width="752" alt="image" src="https://github.com/dotnet/docs/assets/62630498/05db0a7e-a789-4879-af34-0bbae76d5667">

From https://dotnet.microsoft.com/en-us/download/dotnet/8.0, looks like it suggest only the Windows Visual Studio will support .NET 8 but not the macOS version.

From your end, do you know if there will ever be an Visual Studio for Mac 2022 update that will support .NET SDK 8?

Reply:
Unfortunately I don't think so. [What's happening to Visual Studio for Mac?](https://learn.microsoft.com/en-us/visualstudio/mac/what-happened-to-vs-for-mac?view=vsmac-2022) clarifies this. Some key points I pulled from the article:

- Support ends this year on Aug 31st.
- Microsoft doesn't plan to add support for .NET 8 or C#12 to Visual Studio for Mac.
- VSCode is the replacement on Mac.

It seems someone on that team added .NET 8 as a "preview feature" simply because it's planned as not supported but wanted to give it people before VS on Mac died. 😢